packages/Symfony/src/Rector/Class_/MakeCommandLazyRector.php

```diff
@@ -140,16 +168,37 @@ private function resolveCommandNameAndRemove(Class_ $class): ?Node
             }
 
             if ($node instanceof StaticCall) {
```

Comment:
@TomasVotruba Here is last bug :)

With class:
```php
class SomeCommand extends Command
{
    protected function configure()
    {
        $this->setName('some:command');
    }

    protected function execute(InputInterface $input, OutputInterface $output)
    {
        // This will fail
        SomeClass::staticMethodCall();
    }
}
```
It will fail cause that check is global but should be local, adding:
```diff
            if ($node instanceof StaticCall) {
+                if (! $this->isType($node, self::COMMAND_CLASS)) {
+                    return null;
+                }
```
Fixes the issue cause otherwise the `$commandName` was replaced with `null`.

Reply:
Great examples + fix, just copy-pasted and it works :+1:

Reply:
Btw, is this the "randomly (like one on 20/30) classes is not getting correct property being set" bug?